scio-test/src/it/scala/com/spotify/scio/avro/TypedAvroIT.scala

```diff
@@ -23,11 +23,11 @@ import org.scalatest.{FlatSpec, Matchers}
 
 class TypedAvroIT extends FlatSpec with Matchers {
   "Typed Avro IO" should "not throw exception" in {
-    noException should be thrownBy {
+    // noException should be thrownBy {
```

Comment:
Is this failing? Should we add a `FIXME` or something to fix it later?

Reply:
Yeah do not merge just yet. 
I forgot about this one but I need to have a look at it...

Reply:
Fixed. The test was actually passing.